packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -234,6 +234,9 @@ export const defaultErrorMessages = {
       overview: `doesn't apply the "ref" prop to its root DOM node.`,
       suggestions: [
         `Make sure you're applying the ref to the ${resolveInfo('root element')} in your component.`,
+        `Check if your component has a primary slot other than root, and add ${resolveInfo(
```

Comment:
nit/maybe
```suggestion
        `Check if your component has a customized primary slot, and add ${resolveInfo(
```

Reply:
Hmm I agree the wording isn't necessarily the best, but I'm not sure "customized" is the best description. We don't use that term anywhere else in reference to the primary slot. I'll leave it as-is unless you feel strongly about it.

Reply:
"overrides the primary slot" maybe?